scipy/stats/_multivariate.py

```diff
@@ -3029,7 +3030,19 @@ def _process_parameters(self, n, p):
         flagging values out of the domain.
         """
         p = np.array(p, dtype=np.float64, copy=True)
-        p[..., -1] = 1. - p[..., :-1].sum(axis=-1)
+
+        # Was simply:
+#        p[..., -1] = 1. - p[..., :-1].sum(axis=-1)
+        # but this didn't work for rows that sum > 1. It could even fail when
+        # the row summed to exactly 1. See gh-11860.
+        shape = p.shape
+        p = np.atleast_2d(p)
+        psum = np.sum(p, axis=-1, keepdims=True)  # keep last dim to broadcast
+        i_pgt = (psum > 1)[..., 0]   # eliminate last dim to index
+        i_plt = (psum < 1)[..., 0]
+        p[i_pgt, :] /= psum[i_pgt]  # rows that sum > 1 normalize by division
+        p[i_plt, -1:] += 1 - psum[i_plt]  # rows that sum < 1 replace last element
```

Comment:
This is still a back-compat break. Consider:
```
p = [1/8] * 9
```
previously this would result in `p[:8]` having equal `1/8` probability and `p[-1]` being treated as 0. Now it is normalized and everything is treated as equal `1/9` probability.

That said, I think the old behaviour is really surprising so maybe it's worth breaking compatibility and just give a warning or error.

Reply:
If the sum is slightly greater than one, yes. I guess I had restricted my attention to when the last element was small.

You mean just give a warning or error instead of returning NaNs? Why not add normalization as an optional parameter?